packages/date/src/element.ts

```diff
@@ -0,0 +1,113 @@
+import { html, LitElement, property, PropertyValues, TemplateResult } from 'lit-element';
+import { format, formatDistance, formatRelative, Locale } from 'date-fns';
+import { enGB } from 'date-fns/locale';
+
+const defaultDateFormat = 'dd.MM.yyyy';
+const defaultTimeFormat = 'HH:mm';
+const defaultTimeSecondsFormat = 'HH:mm:ss';
+const defaultDateTimeFormat = 'HH:mm dd.MM.yyyy';
+const defaultDateTimeSecondsFormat = 'HH:mm:ss dd.MM.yyyy';
```

Comment:
Should be exported as `enum`, then `variant` can be dropped

Reply:
format has been replaced by string value enum